### DISCUSSION TITLE
How to manage multiple servers from one telegram bot?

### DISCUSSION THREAD

**Discussion by denncom** [Gozargah/Marzban#239] [Q&A]
I installed your panel on several servers in different locations and the question arose how to manage them from one telegram bot?

**Comment by SaintShit** [comment]
> I installed your panel on several servers in different locations and the question arose how to manage them from one telegram bot?

Hi, you'll need to make a bot to connect to all the APIs, or you can just use marzban-node, which is upcoming feature.
